CHANGELOG.rst

```diff
@@ -1,16 +1,54 @@
 Changelog
 =========
 
-Unreleased
+3.0.0 (2019-01-01)
 ------------------
-
+OAuth2.0 Provider - outstanding Features
+
+* OpenID Connect Core support
+* RFC7662 Introspect support
+* RFC8414 OAuth2.0 Authorization Server Metadata support (#605)
+* RFC7636 PKCE support (#617 #624)
+
+OAuth2.0 Provider - Bugfixes
+
+* Add "request" to confirm_redirect_uri #504
+* confirm_redirect_uri/get_default_redirect_uri has a bit changed #445
+* empty scopes no longer raise exceptions for implicit and authorization_code #475 / #406
+* invalid_client is now a FatalError #606
+* Changed errors status code from 401 to 400:
```

Comment:
This should be highlighted as a breaking change.
Clients which for some reason expect this to happen should be aware that this should be adjusted.